apps/emqx_exhook/src/emqx_exhook_mgr.erl

```diff
@@ -162,9 +162,7 @@ pre_config_update(_, {enable, Name, Enable}, OldConf) ->
     case replace_conf(Name,
                       fun(Conf) -> Conf#{<<"enable">> => Enable} end, OldConf) of
         not_found -> {error, not_found};
-        NewConf ->
```

Comment:
Perhaps replace with `?INFO()` ?

Reply:
I think this looks like a debug print that was left behind accidentally, more than something that we really wanted to log.